.github/workflows/cd.yaml

```diff
@@ -0,0 +1,35 @@
+name: cd
+on:
+  pull_request:
```

Comment:
I think we actually want on push to master. Admins can push directly without opening a PR